File: src/main/java/com/elchologamer/userlogin/listener/OnPlayerQuit.java
package com.elchologamer.userlogin.listener;

import com.elchologamer.userlogin.UserLogin;
import org.bukkit.event.EventHandler;
import org.bukkit.event.player.PlayerQuitEvent;

public class OnPlayerQuit extends BaseListener {

    @EventHandler
    public void onPlayerQuit(PlayerQuitEvent e) {
        UserLogin.getPlugin().getPlayer(e.getPlayer()).onQuit();
    }
}

File: src/main/kotlin/com/elchologamer/userlogin/listener/OnPlayerQuit.kt
package com.elchologamer.userlogin.listener

import com.elchologamer.userlogin.UserLogin.Companion.plugin
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerQuitEvent

object OnPlayerQuit : Listener {
    @EventHandler
    fun onPlayerQuit(e: PlayerQuitEvent) {
        plugin.getPlayer(e.player).onQuit()
    }
}